debian/qubes-core-agent.install

```diff
@@ -51,7 +51,7 @@ etc/sudoers.d/umask
 etc/sysctl.d/20_tcp_timestamps.conf
 etc/sysctl.d/80-qubes.conf
 etc/systemd/system/haveged.service
-etc/udev/rules.d/50-qubes-misc.rules
+etc/udev/rules.d/50-qubes-mem-hotplug.rules
```

Comment:
While at it, this should really be in `/lib/udev`, `/etc` is for user configs.

Reply:
done